src/client/wire_protocol/operations.rs

```diff
@@ -181,7 +219,9 @@ impl Message {
         let option_length = match return_field_selector {
             Some(ref bson) => match bson.byte_length() {
                 Ok(i) => i,
-                Err(_) => return Err("Unable to serialize return_field_selector".to_owned())
+                Err(_) =>
+                    return Err("Unable to serialize \
```

Comment:
I'd prefer not to include a line break within a string unless absolutely necessary. In this case, we can fit the string in one line.